projects/fsastorefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -34,9 +34,9 @@ export const storefrontRoutesConfig: RoutesConfig = {
   noClaims: { paths: ['noClaims'] },
   claimsPage: { paths: ['claimsPage'] },
   contactAgent: { paths: ['contact-agent/:agent'] },
-  productActivation: { paths: ['orgUnit/:orgUnitId/productActivation'] },
-  productAssignment: { paths: ['orgUnit/:orgUnitId/productAssignments'] },
-  unitDetails: { paths: ['orgUnit/:orgUnitId'] },
+  // productActivation: { paths: ['organization/:orgUnitId/productActivation'] },
+  // productAssignment: { paths: ['organization/:orgUnitId/productAssignments'] },
+  // unitDetails: { paths: ['organization/:orgUnitId'] },
```

Comment:
Until we fully implement our feature into B2B the routes are commented-out since they clash with the ones used by Spartacus